apps/health_campaign_field_worker_app/lib/pages/beneficiary/household_overview.dart

```diff
@@ -445,10 +445,8 @@ class _HouseholdOverviewPageState
                                                       sideEffectData,
                                                     )
                                                   : false,
-                                              // TODO Need to handle the null check
-                                              name: e.name?.givenName ?? ' - ',
-                                              // name:
-                                              //     '${e.name?.givenName ?? ' - '} ${e.name?.familyName ?? ' - '}',
+                                              name:
+                                                  '${e.name?.givenName ?? ' - '} ${e.name?.familyName ?? ' - '}',
```

Comment:
this will be there in view beneficiary and beneficiary details page as well right is it handled

Reply:
yes